addons/crm_iap_lead/static/src/js/leads_tree_generate_leads.js

```diff
@@ -0,0 +1,66 @@
+odoo.define('crm.leads.tree', function (require) {
+"use strict";
+    var ListController = require('web.ListController');
+    var ListView = require('web.ListView');
+
+    var KanbanController = require('web.KanbanController');
+    var KanbanView = require('web.KanbanView');
+
+    var viewRegistry = require('web.view_registry');
+
+    function renderGenerateLeadsButton() {
```

Comment:
Can you check with the framework if it is possible to add it onlty on the crm kanban view.